lib/linter/apply-disable-directives.js

```diff
@@ -258,17 +333,45 @@ function applyDirectives(options) {
     const unusedDisableDirectivesToReport = options.directives
         .filter(directive => directive.type === "disable" && !usedDisableDirectives.has(directive));
 
-    const processed = processUnusedDisableDirectives(unusedDisableDirectivesToReport);
 
-    const unusedDisableDirectives = processed
+    const unusedEnableDirectivesToReport = new Set(
+        options.directives.filter(directive => directive.unprocessedDirective.type === "enable")
+    );
+
+    if (
+
+        /*
+         * If directives has the eslint-enable directive,
+         * check whether the eslint-enable comment is used.
+         */
```

Comment:
This comment is more helpful above the `if`:

```suggestion
    /*
     * If directives has the eslint-enable directive,
     * check whether the eslint-enable comment is used.
     */
    if (
```